skr/Makefile

```diff
@@ -1,6 +1,10 @@
-
+# To make k3d-registry.localhost:4000/kyma-watcher:latest a valid image tag, you should do the following:
+# 1. Execute `k3d registry create registry.localhost --port=4000 --verbose`
+# 2. Add "127.0.0.1 k3d-registry.localhost" as a separate line in your /etc/hosts file
 # Image URL to use all building/pushing image targets
-IMG ?= jaythedevil666/kyma-watcher:latest
+IMG ?= k3d-registry.localhost:4000/kyma-watcher:latest
```

Comment:
we do not know for sure if users will use a k3d registry, so please make as generic as possible